src/ProjectTemplates/Web.ProjectTemplates/content/RazorComponentsWeb-CSharp/App.razor

```diff
@@ -1,5 +1,8 @@
-﻿@*
-    The Router component displays whichever component has a @page
-    directive matching the current URI.
-*@
-<Router AppAssembly="typeof(Startup).Assembly" />
+﻿<CascadingAuthenticationState>
```

Comment:
I'm proposing we put `<CascadingAuthenticationState>` here in all cases, even NoAuth, since there's no drawback and it will help people if they try to use `<AuthenticationView>` later.